packages/flame/lib/src/components/mixins/draggable.dart

```diff
@@ -5,6 +5,9 @@ import '../../gestures/events.dart';
 import '../base_component.dart';
 
 mixin Draggable on BaseComponent {
+  bool _isDragged = false;
+  bool get isDragged => _isDragged;
```

Comment:
I know you followed my own naming convention, but now I am thinking isBeingDragged might be more clear. not sure though

Reply:
Naah, I think that is too verbose, this and `isHovered` should be fine imho.